test/Rx/Functional/Observable/IteratorObservableTest.php

```diff
@@ -104,6 +104,44 @@ public function generator_dispose()
         ], $results->getMessages());
     }
 
+    /**
+     * @test
+     *
+     * @link https://github.com/ReactiveX/RxPHP/issues/39
+     */
+    public function splObjectStorage_many()
+    {
+
+        if (defined('HHVM_VERSION') && version_compare(HHVM_VERSION, '3.11.0', 'lt')) {
```

Comment:
Does this mean that we only support HHVM at this version or higher? Is there a way to fix the functionality so that it does work in lower versions?